lib/autokey/scripting/window.py

```diff
@@ -355,15 +364,10 @@ def get_window_geom(self, window_id):
         @return: C{[offsetx, offsety, sizex, sizey]} Returns none if no matches are found
         """
 
-        windowList = self._run_wmctrl(["-lG"])[1].split("\n")
-        output = []
-        for window in windowList:
-            w_split= window.split("  ")
-            hexid = w_split[0]
-            geom1 = w_split[1].split(" ")[1:]
-            geom2 = w_split[2].split(" ")
-
-
-            if hexid==window_id:
-                return [geom1+geom2]
-        return None
\ No newline at end of file
+        returncode, output = self._run_wmctrl(["-lG"])
+        regex = r"^(0x[0-9a-fA-F]{8})\s*(\d*)\s*(\d*)\s*(\d{1,})\s*(\d{1,})\s*(\d{1,})\s*(.*?)$"
```

Comment:
Write a comment that breaks this regex down into its components please. Also things like examples of input it expects, and example of match/output. (Doing this in tests would also be a good idea; point people to the test in a comment here).

Also consider making your regexes specific constants, [like is done elsewhere in the codebase](https://github.com/autokey/autokey/blob/4b87b32452cd41d2b98341db220e99cf4f9e34d6/lib/autokey/model.py#L35)